### PR TITLE
Microsoft.OpenSSH version 8.9.1.0

### PR DESCRIPTION
**Pull request by tgauth**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58756)

### CODE REVIEW

File: manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenSSH
+PackageVersion: 8.9.1.0
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

File: manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenSSH
+PackageVersion: 8.9.1.0
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

File: manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenSSH
+PackageVersion: 8.9.1.0
```
Comment (OfficialEsco) [inline manifests/m/Microsoft/OpenSSH/8.9.1.0/Microsoft.OpenSSH.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Ok so, i got some bad news, we cannot accept this version because it will send our clients into a upgrade loop because the DisplayVersion is set to 1.1.0
```
Name          : OpenSSH
Publisher     : Microsoft Corporation
Version       : 1.1.0
ProductCode   : {F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}
Uninstall     : winget uninstall --id "{F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}"
```
Since this is a Microsoft fork and your a Employee ill assume this should be quickly fixed for the next installer.

**Comment by tgauth** [comment]
> Ok so, i got some bad news, we cannot accept this version because it will send our clients into a upgrade loop because the DisplayVersion is set to 1.1.0
> 
> ```
> Name          : OpenSSH
> Publisher     : Microsoft Corporation
> Version       : 1.1.0
> ProductCode   : {F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}
> Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}
> Uninstall     : winget uninstall --id "{F4EEBD1B-C3B1-4DBD-9FAE-26DD816C288D}"
> ```
> 
> Since this is a Microsoft fork and your a Employee ill assume this should be quickly fixed for the next installer.

I updated the package version, per your suggestions, to 1.1.0. 
For future installers, the package version will need to increment from 1.1.0? 
Is there anything else that needs to be fixed?

**Comment by OfficialEsco** [comment]
> Ah, I think I see the issue now - the MSI product version (1.1.0) does not match the actual release version (8.9.1.0)

Yeeeees, i'm willing to bet the software will always be 1.1.0 which means none will be able to upgrade because you have to keep replacing the url/hash.
I didn't actually want you to accept the suggested changes, but if you want it merged you'll have to rename to version folder name by using VSCode online via `.`

**Comment by tgauth** [comment]
> 



> > Ah, I think I see the issue now - the MSI product version (1.1.0) does not match the actual release version (8.9.1.0)
> 
> Yeeeees, i'm willing to bet the software will always be 1.1.0 which means none will be able to upgrade because you have to keep replacing the url/hash. I didn't actually want you to accept the suggested changes, but if you want it merged you'll have to rename to version folder name by using VSCode online via `.`

No, that's ok - I made the changes before I understood the root of the issue. I think we'll address it in the MSI going forward. I'll close this PR, for now. Thanks!

**Pull request closed without merging**